lib/content-services/src/lib/tag/tag-list.component.html

```diff
@@ -1,6 +1,6 @@
 <mat-chip-list class="adf-tag-chips-list">
     <div class="adf-list-tag" *ngFor="let currentEntry of tagsEntries; let idx = index">
-        <mat-chip class="adf-primary-background-color">
+        <mat-chip class="adf-primary-background-color" *ngIf="currentEntry.entry.count">
```

Comment:
Might be better checking for entry existence:
```suggestion
        <mat-chip class="adf-primary-background-color" *ngIf="currentEntry.entry?.count">
```

Reply:
Are you sure that the API can't return a `count=0`?